packages/shared/src/components/Feed.spec.tsx

```diff
@@ -675,9 +674,6 @@ it('should report broken link with comment', async () => {
   contextBtn.click();
   const brokenLinkBtn = await screen.findByText('Broken link');
   brokenLinkBtn.click();
-  const input = (await screen.findByRole('textbox')) as HTMLTextAreaElement;
-  fireEvent.change(input, { target: { value: 'comment' } });
-  input.dispatchEvent(new Event('input', { bubbles: true }));
```

Comment:
This needs to be reverted because of https://github.com/dailydotdev/apps/pull/1941#discussion_r1259787245 right?